app/views/layouts/admin/layout.html.haml

```diff
@@ -2,14 +2,15 @@
 %html
   %head
     %meta{charset: "utf-8"}
-    %title TfgTemplate: Admin
+    %title= page_title
```

Comment:
strongly disagree. For all intents and purposes I want us to view the admin section as an entire app buried inside the other app.

It should not share any view/style/js as much as possible so that we can keep it modular and updated independently of the rest of the app. That way children of RT can continually get improved functionality from their admin site without having to change the rest of the app.